File: js/slider.js
document.addEventListener("DOMContentLoaded", () => {

    const slides = document.querySelectorAll(".hero-slider .slide");

    if(slides.length > 0){
        let current = 0;

        setInterval(() => {

            slides[current].classList.remove("active");

            current = (current + 1) % slides.length;

            slides[current].classList.add("active");

        }, 5000);
    }

});

File: slider.java
const slides = document.querySelectorAll(".hero-slider .slide");
let index = 0;

function showSlide(n){
    slides.forEach(slide => slide.classList.remove("active"));
    slides[n].classList.add("active");
}

function nextSlide(){
    index = (index + 1) % slides.length;
    showSlide(index);
}

showSlide(0);
setInterval(nextSlide, 5000);